webapp/src/js/services/report-view-model-generator.js

```diff
@@ -96,7 +102,17 @@ angular.module('inboxServices').factory('ReportViewModelGenerator',
               if (summaries && summaries.length) {
                 model.formatted.subject = summaries.pop().subject;
               }
-
+              if (_.isArray(model.formatted.fields.data)) {
+                const labels = ['Medic ID', 'Name'];
```

Comment:
This is using the translated value so again it won't work unless you're in English and haven't changed from the default translation. I think it would be better to make this change in the FormatDataRecord service where you have access to the label so you can check that instead of the translated value.

If you're careful you can make changes in FormatDataRecord so xml and json reports both set a `url` property which will make the template a little cleaner because it'll all be handled the same way.

Let me know if you need a hand working on the FormatDataRecord service - it's not the nicest code...